scripts/core/menu/views/about.html

```diff
@@ -10,9 +10,9 @@ <h3 class="modal-header__sd-logo">Superdesk</h3>
     <div class="about-content">
       <span class="sf-logo"></span>
       <h4>Superdesk {{ :: version }}</span></h4>
-      <h5 ng-if="build_rev && build_rev !== ''">Superdesk rev. {{ :: build_rev }}</h5>
-      <h5 ng-if="build_rev_core && build_rev_core !== ''">Superdesk core rev. {{ :: build_rev_core }}</h5>
-      <h5 ng-if="build_rev_client && build_rev_client !== ''">Superdesk client rev. {{ :: build_rev_client }}</h5>
+      <h5 ng-if="build_rev && build_rev !== ''">Superdesk rev. <a href="https://github.com/superdesk/superdesk/commit/{{ :: build_rev }}" rel="noopener noreferrer">{{ :: build_rev }}</a></h5>
+	  <h5 ng-if="build_rev_core && build_rev_core !== ''">Superdesk core rev. <a href="https://github.com/superdesk/superdesk-core/commit/{{ :: build_rev_core }}" rel="noopener noreferrer">{{ :: build_rev_core }}</a></h5>
```

Comment:
yes indeed thanks! Dunno why test are failing, seems to be always the same thinks, delay make them false